core/src/main/scala/com/microsoft/azure/synapse/ml/io/http/RESTHelpers.scala

```diff
@@ -89,7 +94,18 @@ object RESTHelpers {
   }
 
   def parseResult(result: CloseableHttpResponse): String = {
-    IOUtils.toString(result.getEntity.getContent, "utf-8")
+    var res: String = ""
```

Comment:
dont use var just return the result

Reply:
Please ignore any changes to this file "RESTHelpers". I will revert them in next iteration. The change seen was meant for debugging.